CONTRIBUTING.md

```diff
@@ -82,6 +82,4 @@ Our code is our lifeblood so maintaining Cesium's high code quality is important
 
 # Code of Conduct
 
-One of Cesium's strengths is our community.  Our contributors and users are pushing the 3D geospatial field to amazing new levels.  We rely on an open, friendly, inclusive environment to facilitate this.  As such, we follow the [Contributor Covenant](http://contributor-covenant.org/)'s [Code of Conduct](http://contributor-covenant.org/version/1/4/code_of_conduct.md) to ensure a harassment-free experience in the Cesium community.  Any unacceptable behavior can be confidentially sent to the core team at pcozzi@agi.com.
-
-This applies to the main Cesium repo, forum, twitter, and all channels, including all repos in the [AnalyticalGraphicsInc](https://github.com/AnalyticalGraphicsInc) GitHub organization.
+Cesium's Code of Conduct is [here](./CODE_OF_CONDUCT.md).
```

Comment:
Can we make this a duplicate of the `To ensure an inclusive community, contributors and users in the Cesium community should follow the [code of conduct]` line? It just reads better.

Reply:
@ggetz I'll work on doing that.